<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1498</width>
    <height>997</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>2D Triangulation on 3D Sphere</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="glViewer" name="m_glviewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1498</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuGeneration">
    <property name="title">
     <string>Generation</string>
    </property>
    <addaction name="actionPoints"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>Function</string>
    </property>
    <addaction name="actionConvexHull"/>
    <addaction name="actionConvexHull_EE"/>
    <addaction name="actionConvexHull_Jarvis_March"/>
    <addaction name="actionConvexHull_Graham_Scan"/>
    <addaction name="actionConvexHull_Divide_and_Conquer"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuGeneration"/>
   <addaction name="menuFunction"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_info">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1400</width>
     <height>300</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QTextBrowser" name="m_information">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1491</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="text">
    <string>Points</string>
   </property>
  </action>
  <action name="actionConvexHull">
   <property name="text">
    <string>ConvexHull_EP</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionRepaint">
   <property name="text">
    <string>Repaint</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionConvexHull_EE">
   <property name="text">
    <string>ConvexHull_EE</string>
   </property>
  </action>
  <action name="actionConvexHull_Jarvis_March">
   <property name="text">
    <string>ConvexHull_Jarvis_March</string>
   </property>
  </action>
  <action name="actionConvexHull_Graham_Scan">
   <property name="text">
    <string>ConvexHull_Graham_Scan</string>
   </property>
  </action>
  <action name="actionConvexHull_Divide_and_Conquer">
   <property name="text">
    <string>ConvexHull_Divide_and_Conquer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glViewer</class>
   <extends>QWidget</extends>
   <header>glViewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
